### PR TITLE
[4.x] render image after get size

### PR DESCRIPTION
**Pull request by sonnt612**
### Checks

- [x] I have read the contribution guidelines regarding Pull Requests here: https://git.io/JJ0Pg

### CODE REVIEW

**Comment by jsamr** [comment]
@sonnt612 Thanks for your contribution! The appropriate branch to merge this PR into would be `dev/4.x`, would be nice if your could resubmit.

**Comment by sonnt612** [comment]
> @sonnt612 Thanks for your contribution! The appropriate branch to merge this PR into would be `dev/4.x`, would be nice if your could resubmit.

ok. I changed to branch dev/4.x

File: src/HTMLImage.js [hunk 1]
```diff
@@ -103,13 +105,17 @@ export default class HTMLImage extends PureComponent {
...
 
     validImage (source, style, props = {}) {
-        return (
-            <Image
-              source={source}
-              style={[style, { width: this.state.width, height: this.state.height, resizeMode: 'cover' }]}
-              {...props}
-            />
-        );
+        if (this.state.loading) {
```
Comment (jsamr) [inline src/HTMLImage.js:108]: I would have thought you would render the image if it is *not* loading?

**Review by jsamr — CHANGES_REQUESTED** [review]
See my comments

**Comment by jsamr** [comment]
I'm closing because this issue has been resolved in v5.0.0 which has a very easy upgrade path from v4.x. See #442

**Pull request closed without merging**